internal/nexodus/nexodus.go

```diff
@@ -452,8 +493,8 @@ func (ax *Nexodus) Start(ctx context.Context, wg *sync.WaitGroup) error {
 		for _, proxy := range ax.egressProxies {
 			proxy.Start(ctx, wg, ax.userspaceNet)
 		}
-
```

Comment:
add your first security group reconcile here (or a few lines above right before / after doing the first device reconcile)